main.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from backend.app import create_app
+from backend.extensions import db
+from backend.model import Incident
+from typing import List, Dict
+
+
+def main():
+    app = create_app()
+
+    with app.app_context():
+        db.create_all()
+
+    write_to_db(app)
+
+def get_data(json_file: str) -> List[Dict]:
+    with open(json_file, 'r') as data_file:
+        data = json.loads('\n'.join(data_file.readlines()))
+        return data['data']
+
+def write_to_db(app):
+    with app.app_context():
+        data = get_data('../police-brutality-data.json')
+        for i in range(10):
```

Comment:
Does this only add 10 records to the db?

Reply:
Yes, only 10 for testing purposes

Reply:
sounds good